src/com/palmergames/bukkit/towny/huds/HUDManager.java

```diff
@@ -143,8 +146,41 @@ else if (e.getTownBlock() != null)
 				} catch (Exception ex) {}
 	}
 
-	//**UTILS**//
-	public static String check(String check) {
-		return check.length() > 16 ? check.substring(0, 16) : check;
+	public static String check(String string) {
+		return string.length() > 32 ? string.substring(0, 32) : string;
```

Comment:
Bump this up to either 128 or 64. It was just recently bumped up to 128 by another PR.